app.js

```diff
@@ -9,11 +8,71 @@ class Dashboard extends React.Component {
     /**
      * @inheritdoc
      */
+    constructor(props) {
+        super(props);
+        this.psnrTestResultTransformFunc = jsonData => {
+            const extractData = (json, yAxisFieldName) =>
+                json.reduce((currData, currRow) => {
+                    const field = currRow[yAxisFieldName];
+
+                    currData.push([ currRow.buildNum, field ]);
```

Comment:
Is it a problem that you're modifying the passed array in place? I don't see it being a problem...but I want to be sure because that's caused problems for me in other codebases.

Reply:
isn't this always the case with `reduce` ?

Reply:
I'm referring specifically to the currData.push.